test/PushComm/unit_tests/SubscribeBySig/SubscribeBySig.t.sol

```diff
@@ -0,0 +1,294 @@
+pragma solidity ^0.8.0;
+
+import { BaseTest } from "../../../BaseTest.t.sol";
+import { CoreTypes } from "../../../../contracts/libraries/DataTypes.sol";
+import { SignatureVerifier } from "contracts/mocks/MockERC1271.sol";
+
+import "forge-std/console.sol";
+
+contract SubscribeBySig_Test is BaseTest {
+    bytes constant _testChannelIdentity = bytes("test-channel-hello-world");
+
+    event Subscribe(address indexed channel, address indexed user);
+    event Unsubscribe(address indexed channel, address indexed user);
+
+    SignatureVerifier verifierContract;
+
+    function setUp() public override {
+        BaseTest.setUp();
+
+        changePrank(actor.tim_push_holder);
+        verifierContract = new SignatureVerifier();
+
+        changePrank(actor.bob_channel_owner);
+        coreProxy.createChannelWithPUSH(CoreTypes.ChannelType.InterestBearingOpen, _testChannelIdentity, 50e18, 0);
+    }
+
+    modifier whenUserSubscribesWith712Sig() {
+        _;
+    }
+
+    function testFail_WhenMaliciousUserReplaysA712Signature() public whenUserSubscribesWith712Sig {
```

Comment:
is this issue of expectRevert remaining?
and if not, then I think we should remove testFail and use expectRevert, as it's the case in all tests